Source/JavaScriptCore/wasm/js/WebAssemblyFunction.h

```diff
@@ -92,6 +92,11 @@ class WebAssemblyFunction final : public WebAssemblyFunctionBase {
     // to our Instance, which points to the Module that exported us, which
     // ensures that the actual Signature/code doesn't get deallocated.
     CodePtr<WasmEntryPtrTag> m_jsEntrypoint;
+    // This is the callee needed by LLInt/IPInt
+    // This keeps it alive
+    RefPtr<Wasm::Callee> m_wasmCallee;
```

Comment:
1) Does this need to keep the LLInt callee alive?
2) If so, is this going to be a problem? Should we clear this out when we tier up?

Reply:
I'll remove this